allennlp/data/batch.py

```diff
@@ -5,9 +5,9 @@
 
 import logging
 from collections import defaultdict
-from typing import Dict, List, Union, Iterator, Iterable
+from typing import Dict, Iterable, Iterator, List, Union
 
-import numpy
+import numpy as np
```

Comment:
Please don't do this.  If you're writing code that uses numpy, I won't complain if you import it like this.  But don't go changing all of the imports in the code to use this contraction.